src/api/axiosInstance.js

```diff
@@ -17,9 +17,13 @@ import $ from 'jquery';
  *
  * @module Axios
  */
-const createRequestInstance = (resolve) => {
+const createRequestInstance = (resolve, timeout) => {
```

Comment:
Update the docs. Is it seconds or milliseconds?

Reply:
Done